editor.planx.uk/src/@planx/components/List/schemas/Zoo.ts

```diff
@@ -0,0 +1,69 @@
+import { TextInputType } from "@planx/components/TextInput/model";
+
+import { Schema } from "../model";
+
+/**
+ * Temp simple example to build out UI
+ * Can be re-used as mock for testing
+ */
+export const Zoo: Schema = {
+  type: "Animal",
+  fields: [
+    // Text - short
+    {
+      type: "text",
+      data: {
+        title: "What's their name?",
+        description: "Please make it cute",
+        fn: "name",
+        type: TextInputType.Short,
+      },
+    },
+    // Text - email
+    {
+      type: "text",
+      data: {
+        title: "What's their email address?",
+        fn: "email",
+        type: TextInputType.Email,
+      },
+    },
+    // Number
+    {
+      type: "number",
+      data: {
+        title: "How old are they?",
+        fn: "age",
+        units: "years old",
+        allowNegatives: false,
+      },
+    },
+    // Question - multiple options
+    {
+      type: "question",
+      data: {
+        title: "What size are they?",
+        fn: "size",
+        options: [
+          { id: "small", data: { text: "Small" } },
+          { id: "medium", data: { text: "Medium" } },
+          { id: "large", data: { text: "Large" } },
+        ],
+      },
+    },
+    // Question - only two options
+    {
+      type: "question",
+      data: {
+        title: "How cute are they?",
+        fn: "cuteness",
+        options: [
+          { id: "very", data: { text: "Very" } },
+          { id: "super", data: { text: "Super" } },
+        ],
+      },
+    },
+  ],
+  min: 1,
+  max: 10,
+} as const;
```

Comment:
schema definition files feel like a good MVP route here & editable by someone like August in very near future until more general editor customisation is possible :heavy_plus_sign: